lib/grape/validations/validators/base.rb

```diff
@@ -10,10 +10,11 @@ class Base
       # @param options [Object] implementation-dependent Validator options
       # @param required [Boolean] attribute(s) are required or optional
       # @param scope [ParamsScope] parent scope for this Validator
-      def initialize(attrs, options, required, scope)
+      def initialize(attrs, options, required, fail_fast, scope)
```

Comment:
@dblock I've thought about it. It could be a part of the scope, but I am not sure if it is the best place for validation options. Thoughts?

I've changed the constructor to the following to keep existing custom validators working.
`def initialize(attrs, options, required, fail_fast, scope, opts = {})`